packages/bpk-component-modal/src/BpkModalDialog.js

```diff
@@ -45,6 +45,11 @@ const BpkModalDialog = props => {
 
   const headingId = `bpk-modal-heading-${props.id}`;
 
+  const headerClassNames = [getClassName('bpk-modal__header')];
+  if (!props.title) {
```

Comment:
`title` is a required prop therefore it should never be empty

Reply:
I suggest that we always provide a title for screen readers, but just hide it visually. Most of the examples in the linked card include content that can be extracted out to a title.

Reply:
Begs the question why we would want to hide the title in the first place.

Reply:
In the design there are dialogs without title, should I just hide it then?